<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInbox</class>
 <widget class="QDialog" name="UserInbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inbox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QListWidget" name="messageListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="emailDisplayEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="replyGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Reply</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="replyButton">
          <property name="text">
           <string>Reply</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/32/Reply_Message</normaloff>:/icons/32/Reply_Message</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="toolBarListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Inbox</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/resources.qrc">
        <normaloff>:/icons/32/Email_Tab</normaloff>:/icons/32/Email_Tab</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sent Mail</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/resources.qrc">
        <normaloff>:/icons/32/Sent_Messages</normaloff>:/icons/32/Sent_Messages</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Mail</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/resources.qrc">
        <normaloff>:/icons/32/New_Message</normaloff>:/icons/32/New_Message</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Delete Mail</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/resources.qrc">
        <normaloff>:/icons/32/Delete_Mail</normaloff>:/icons/32/Delete_Mail</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Send/Recieve</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/resources.qrc">
        <normaloff>:/icons/32/Send_Recieve</normaloff>:/icons/32/Send_Recieve</iconset>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
